src/types/enhancers.ts

```diff
@@ -0,0 +1,197 @@
+import PropTypes from 'prop-types'
+import * as CSS from 'csstype'
+
+type CssProps = Pick<
+  CSS.StandardProperties,
+  | 'alignContent'
```

Comment:
I might be wrong, but it sounds like this could have been inherited from the enhancers themselves?

Reply:
Aren't these already in the CSS.StandardProperties? Or maybe this fixes an intellisense glitch? @mshwery

Reply:
It could be, and we should do that in a follow up PR. I wanted this in here for bare-minimum types that were still accurate.

Reply:
CSS.StandardProperties was excessive. I think this limits the scope of all possible css properties to only the ones that ui-box explicitly supports.

Reply:
> CSS.StandardProperties was excessive. I think this limits the scope of all possible css properties to only the ones that ui-box explicitly supports.

Gotcha. Thanks!